app/controllers/PromoLandingPage.scala

```diff
@@ -1,45 +1,66 @@
+import views.support.PegdownMarkdownRenderer
 
 object PromoLandingPage extends Controller {
 
   val tpBackend = TouchpointBackend.Normal
   val catalog = tpBackend.catalogService.unsafeCatalog
   val edition = DigitalEdition.UK
 
-  def render(promoCodeStr: String) = CachedAction { implicit request =>
-    val evaluateStarts = tpBackend.environmentName == "PROD"
-    val promoCode = PromoCode(promoCodeStr)
+  private def isActive(promotion: AnyPromotion): Boolean = {
+    val isTest = tpBackend.environmentName != "PROD"
+    isTest || (promotion.starts.isBeforeNow && !promotion.expires.exists(_.isBeforeNow))
+  }
 
+  private def getDigitalPackLandingPage(promotion: AnyPromotion)(implicit promoCode: PromoCode): Option[Html] = {
+    promotion.asDigitalPack.filter(p => isActive(asAnyPromotion(p))).map { promotionWithLandingPage =>
+      views.html.promotion.digitalpackLandingPage(edition, catalog, promoCode, promotionWithLandingPage, PegdownMarkdownRenderer)
+    }
+  }
+
+  private def getNewspaperLandingPage(promotion: AnyPromotion)(implicit promoCode: PromoCode): Option[Html] = {
+    promotion.asNewspaper.filter(p => isActive(asAnyPromotion(p))).map { promotionWithLandingPage =>
+      views.html.promotion.newspaperLandingPage(catalog, promoCode, promotionWithLandingPage, PegdownMarkdownRenderer)
+    }
+  }
+
+  def render(promoCodeStr: String) = CachedAction { _ =>
+    implicit val promoCode = PromoCode(promoCodeStr)
     (for {
       promotion <- tpBackend.promoService.findPromotion(promoCode)
-      promotionWithLandingPage <- promotion.asDigipack
-      html <- if ((evaluateStarts && promotionWithLandingPage.starts.isAfterNow) || promotionWithLandingPage.expires.exists(_.isBeforeNow))
-        None
-      else
-        Some(views.html.promotion.landingPage(edition, catalog.digipack.month, promoCode, promotionWithLandingPage, Config.Zuora.paymentDelay, PegdownMarkdownRenderer))
+      html <- getNewspaperLandingPage(promotion) orElse getDigitalPackLandingPage(promotion)
     } yield Ok(html)).getOrElse(Redirect("/" ? (internalCampaignCode -> s"FROM_P_${promoCode.get}")))
   }
 
   def preview() = GoogleAuthenticatedStaffAction { implicit request =>
     Form(Forms.single("promoJson" -> Forms.text)).bindFromRequest().fold(_ => NotFound, { jsString =>
-      Json.fromJson[AnyPromotion](Json.parse(jsString)).asOpt.flatMap(_.asDigipack)
-      .map(p => views.html.promotion.landingPage(edition, catalog.digipack.month, p.codes.headOption.getOrElse(PromoCode("")), p, Config.Zuora.paymentDelay, PegdownMarkdownRenderer))
-      .fold[Result](NotFound)(p => Ok(p).withHeaders(HandleXFrameOptionsOverrideHeader.HEADER_KEY -> s"ALLOW-FROM ${Config.previewXFrameOptionsOverride}"))
+      Json.fromJson[AnyPromotion](Json.parse(jsString)).asOpt.flatMap(_.asDigitalPack)
+        .map(p => views.html.promotion.digitalpackLandingPage(edition, catalog, p.codes.headOption.getOrElse(PromoCode("")), p, PegdownMarkdownRenderer))
```

Comment:
Yup, I've created a story to tackle that after - the Acquisition team are happy with that.

https://trello.com/c/6cDjeShh/62-enable-previewing-for-newspaper-landing-page